data/settings/src/main/java/com/google/jetpackcamera/settings/LocalSettingsRepository.kt

```diff
@@ -50,7 +51,17 @@ class LocalSettingsRepository @Inject constructor(
                     null -> FlashModeStatus.OFF
                 },
                 front_camera_available = it.frontCameraAvailable,
-                back_camera_available = it.backCameraAvailable
+                back_camera_available = it.backCameraAvailable,
+                aspect_ratio = when (it.aspectRatioStatus) {
+                    AspectRatioProto.ASPECT_RATIO_NINE_SIXTEEN -> AspectRatio.NINE_SIXTEEN
+                    AspectRatioProto.ASPECT_RATIO_ONE_ONE -> AspectRatio.ONE_ONE
+                    // defaults to 3:4 aspect ratio
+                    AspectRatioProto.ASPECT_RATIO_THREE_FOUR,
+                    AspectRatioProto.UNRECOGNIZED,
+                    null -> AspectRatio.THREE_FOUR
```

Comment:
nit: We should make this a function on `model.AspectRatoi`, it'll help make this area of the code cleaner when we add more settings.

```
def fromProto(proto: AspectRatioProto) = when...
```